src/test/common/fdc3.app-channels.ts

```diff
@@ -133,14 +157,14 @@ export function createAppChannelTests(cc: ChannelControl<any, any, any>, documen
       });
     });
 
-    const acTestId10 = "(" + prefix + "ACContextHistoryMultiple) Should retrieve the last broadcast context item when app B broadcasts a context with multiple history items to the same app channel and A gets current context";
-    it(acTestId10, async () => {
+    const acTestId11 = "(" + prefix + "ACContextHistoryMultiple) Should retrieve the last broadcast context item when app B broadcasts a context with multiple history items to the same app channel and A gets current context";
```

Comment:
@robmoffat @gaganahluwalia this description is quite conformance framework centric (history items is not a term defined in FDC3) and might be confusing. How about:

> `Channel.getCurrentContext` should retrieve the last broadcast context item of a specified type when app B broadcasts multiple contexts of that type to the same app channel.

I think the final step of the reproduction instruction is missing (what is received).

I realise this is after the merge, but think these descriptions could/should be less obtuse.